src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
-    .title-banner {
-        &.wbb-bg {
-            background-image: url("/images/hoc2015/hide-bg.jpg");
+    .img,
+    .button {
+        &.card-info {
+            width: calc(100% - 20px);
         }
+    }
 
-        &.dance-bg {
-            background-image: url("/images/hoc2015/dance-bg.jpg");
+    .img {
+        &.card-info {
+            margin: 10px 10px 5px 10px;
+            border-radius: 5px;
         }
 
-        &.name-bg {
-            background-image: url("/images/hoc2015/name-bg.jpg");
+        &.mod-border {
+            border-radius: 5px;
+            width: 100%;
         }
 
-        section {
-            border: 0;
-            padding: 10px 0;
-            max-width: $desktop;
+        &.mod-resource,
+        &.mod-studio {
+            margin-right: 15px;
```

Comment:
This seems like it might be handled better by a wrapper component rather than a modifier if it's for formatting things specific to the HOC view.